app/public.js

```diff
@@ -121,28 +130,21 @@
   }
 
   function styleElement(element, image) {
-    element.style.width = image.width + 'px';
-    element.style.height = image.height + 'px';
+    element.style.width = image.width + 1 + 'px';
+    element.style.height = image.height + 1 + 'px';
     element.style.top = image.position.top + 'px';
     element.style.left = image.position.left + 'px';
-    element.classList.add(getBackgroundColor(getImageCoverage(image)));
+    element.style.backgroundColor = getBackgroundColor(getImageCoverage(image));
+    element.style.border = '1px solid black';
     element.classList.add(OVERLAY_CLASS);
   }
 
   function getBackgroundColor(percentage) {
-    if (percentage >= 75 && percentage < 150) {
-      return COLORS.green;
-    }
-
-    if (percentage >= 150 && percentage < 300) {
-      return COLORS.orange;
-    }
-
-    if (percentage >= 300) {
-      return COLORS.red;
-    }
-
-    return  COLORS.blue;
+    let col = percentage;
+    if (percentage < 0) col = 0;
```

Comment:
@simonhearne CONSISTENCY - always use `{` for any if statements even if it's a single line